orion/config.go

```diff
@@ -73,6 +73,8 @@ type ZipkinConfig struct {
 type NewRelicConfig struct {
 	APIKey            string
 	ServiceName       string
+	//HttpTxNameType decides the transaction name logged in NR. Options are "method" (default), "fullmethod", "url", "route". Default is "method"
```

Comment:
I would suggest to use `fullmethod` as the default since it is same behaviour to [gRPC endpoints](https://github.com/carousell/Orion/blob/07d546eaaf9d99283cb9868b81e440755b1ad3e6/interceptors/interceptors.go#L107). I am also good also to have this as the only option in order to make it consistent across all Orion.
`url` case should not be used, since it makes observability harder.

Reply:
Okay to me using `fullmethod` as the default. Will fire another PR to change it.

I feel a valid case (though rare) in using `route` over `method` or `fullmethod` for HTTP requests: Let's say one service exposes the same endpoint in both HTTP and GRPC. Then both traffic will end up being served by the same service method. If we use the same metric name for both, it is not distinguishable on NR.